python/labbox_ephys/__init__.py

```diff
@@ -18,7 +18,6 @@
 from .misc import get_recording_info, get_recording_object, get_sorting_info, get_unit_waveforms_from_snippets_h5
 from .register_views import (_recording_views, register_recording_view,
                              register_sorting_view, _sorting_views)
-from .sorters import sorters
```

Comment:
For now the way to import sorters is:

`from labbox_ephys import sorters`

and then

```
sorters.mountainsort4(...)
```

This works because sorters is a subdirectory of `labbox_ephys`